app/src/main/java/com/example/concordia_campus_guide/Helper/Notification.java

```diff
@@ -0,0 +1,62 @@
+package com.example.concordia_campus_guide.Helper;
+
+import android.content.Context;
+import android.os.Handler;
+
+import com.example.concordia_campus_guide.Activities.MainActivity;
+import com.example.concordia_campus_guide.Database.AppDatabase;
+import com.example.concordia_campus_guide.Models.CalendarEvent;
+import com.example.concordia_campus_guide.Models.Helpers.CalendarViewModel;
+import com.example.concordia_campus_guide.Models.Place;
+import com.example.concordia_campus_guide.Models.RoomModel;
+
+
+public class Notification {
+    private MainActivity mainActivity;
+    private AppDatabase appDatabase;
+
+    public Notification(MainActivity mainActivity, AppDatabase appDatabase){
+        this.mainActivity =  mainActivity;
+        this.appDatabase  = appDatabase;
+    }
+
+    public void checkUpCalendarEvery5Minutes() {
```

Comment:
Can you reduce this time to check calendar? Make it like every 10 seconds

Reply:
This should be in a global state which we refresh every 10 seconds because both the notification and the search uses it

Reply:
hmm yeah I agree but keep the pop up to 5mins I think it would be annoying if the popup came up every 10 seconds

Reply:
I feel like 10 seconds is too short. Maybe we can reduce it to 5 minutes the check up if guys want. I will do this in a different pr though.

Reply:
We should do this in this PR because this feature is buggy and I don't think we should merge buggy features to master

Reply:
Ok I think it works